pokemon-checker/src/components/PokemonDisplay/PokemonDisplay.tsx

```diff
@@ -21,31 +21,39 @@ type displayProps = {
 
 export const PokemonDisplay = (props: displayProps) => {
   const [pokeonObject, setPokemonObject] = useState<PokemonDTO>();
-  //TODO (jeremy): Move this factory to a service! Views shouldn't control this.
-  const pokemonFactory: PokemonFactory = new PokemonFactory();
-
-  useEffect(() => {
-    fetchPokemonObject();
-  });
+  // TODO (jeremy): Elminate the factory declaration! The service method is
+  // passed to this component as a prop.
+  const pokemonFactory: PokemonFactory = useMemo(
+    () => new PokemonFactory(),
+    []
+  );
+  /**
+   * Create a PokemonObject from the results retrieved
+   * @param pokemonRetrieved the pokemon retrieved from the API
+   */
+  const createPokemonObject = useCallback(
```

Comment:
For a similar reason to memo, [useCallback](https://reactjs.org/docs/hooks-reference.html#usecallback) saves the function declaration so it doesn't have to be re-defined every time a component renders.

This was also included because ESLint was yelling at me.